lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -621,6 +629,13 @@ def write(chunk)
           severity = compute_severity(
             entry_resource.type, record, entry_common_labels)
 
+          # Get trace resource for LogEntry "trace" field per specified config.
```

Comment:
Done. Switched to `fq_traced_id` for fully-qualified trace id and changed the comment accordingly.